dev/com.ibm.ws.install.featureUtility/src/com/ibm/ws/install/featureUtility/cli/FeatureDownloadAction.java

```diff
@@ -0,0 +1,125 @@
+package com.ibm.ws.install.featureUtility.cli;
+
+import java.io.IOException;
+import java.io.PrintStream;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.logging.Level;
+import java.util.logging.Logger;
+
+import com.ibm.ws.install.InstallException;
+import com.ibm.ws.install.featureUtility.FeatureUtility;
+import com.ibm.ws.install.featureUtility.FeatureUtilityExecutor;
+import com.ibm.ws.install.internal.InstallLogUtils;
+import com.ibm.ws.install.internal.InstallLogUtils.Messages;
+import com.ibm.ws.kernel.boot.ReturnCode;
+import com.ibm.ws.kernel.boot.cmdline.ActionHandler;
+import com.ibm.ws.kernel.boot.cmdline.Arguments;
+import com.ibm.ws.kernel.boot.cmdline.ExitCode;
+
+public class FeatureDownloadAction implements ActionHandler {
+
+	private FeatureUtility featureUtility;
+    private Logger logger;
+	private ArrayList<String> artifactNames;
+	private ArrayList<String> artifactShortNames;
+	private List<String> argList;
+	private String location;
+	private boolean isOverwrite;
+
+	@Override
+    public ExitCode handleTask(PrintStream stdout, PrintStream stderr, Arguments args) {
+    	ExitCode rc = initialize(args);
+        if (!!!rc.equals(ReturnCode.OK)) {
+            return rc;
+        }
+        rc = execute();
+        return rc;
+    }
+
+	private ExitCode initialize(Arguments args) {
+		ExitCode rc = ReturnCode.OK;
+		
+		this.logger = InstallLogUtils.getInstallLogger();
+		this.artifactNames = new ArrayList<String>();
+		this.artifactShortNames = new ArrayList<String>();
+		this.argList = args.getPositionalArguments();
+		this.location = args.getOption("location");
+        this.isOverwrite = args.getOption("overwrite") != null;
+        
+        if (location == null) {
```

Comment:
location can be null.  you will download to .m2 if the location is not provided.